samples/Balosar/Balosar.Client/Pages/Counter.razor

```diff
@@ -1,4 +1,5 @@
 ﻿@page "/counter"
+@attribute [Authorize(Roles = "Admin")]
```

Comment:
Should we add a comment indicating that these authorization checks execute client-side - which means they can be bypassed - to encourage folks not to store sensitive stuff in their Razor pages?

Reply:
That's a really good suggestion! I'm pushing the changes in a bit